.github/labeler.yml

```diff
@@ -35,3 +35,8 @@ labels:
   - label: "Bug" # add bug Label to every issue but to no PR
     negate: True
     type: "pull_request"
+  - label: "Internal" # add internal Label to everything that is created by a member of the organization

```

Comment:
right now we add the label to issues & PRs, should we scope it only to PRs?

Reply:
I can see it also being useful for issues, or at least see no harm in adding it there